pkg/util/enc_test.go

```diff
@@ -153,5 +153,13 @@ var _ = Describe("TryToDecodeBase64Gzip", func() {
 			})
 		})
 	})
+})
 
+var _ = Describe("EncodeGzipBase64", func() {
+	It("Encodes a string correctly", func() {
+		input := "HelloWorld"
+		output, err := util.EncodeGzipBase64(input)
+		Expect(err).ShouldNot(HaveOccurred())
```

Comment:
IIRC, we moved to using `ToNot` from `ShouldNot` for errors.  I think it will be good to be consistent.